tests/test_examples.py

```diff
@@ -3,117 +3,310 @@
+    exampletest_single("google2016_unbreakable_0")
+
+
+def test_google2016_unbreakable_1():
+    exampletest_single("google2016_unbreakable_1")
+
+
+def test_grub():
+    exampletest_single("grub")
+
+
+def test_mma_howtouse():
+    exampletest_single("mma_howtouse")
+
+
+@slow_test
+def test_mma_simplehash():
+    exampletest_single("mma_simplehash")  # ~hour
+
+
+def test_securityfest_fairlight():
+    exampletest_single("securityfest_fairlight")
+
+
+def test_strcpy_find():
+    exampletest_single("strcpy_find")
+
+
+def test_whitehat_crypto400():
+    exampletest_single("whitehat_crypto400")
+
+
+def test_whitehatvn2015_re400():
+    exampletest_single("whitehatvn2015_re400")
+
+
+@slow_test
 @flaky(max_runs=3, min_passes=1)
-def test_secconquals2016_ropsynth(): exampletest_single('secconquals2016_ropsynth') # technically not that slow but impossible to run under multiprocessing
-@attr(speed='slow')
-def test_0ctf_momo_3(): exampletest_single('0ctf_momo_3') # 16m
-def test_defcon2016quals_baby_re(): exampletest_single('defcon2016quals_baby-re')
-def test_simple_heap_overflow(): exampletest_single('simple_heap_overflow')
-def test_csci_0a(): exampletest_single('CSCI-4968-MBE/challenges/crackme0x00a')
-def test_csci_1(): exampletest_single('CSCI-4968-MBE/challenges/crackme0x01')
-def test_csci_2(): exampletest_single('CSCI-4968-MBE/challenges/crackme0x02')
-def test_csci_3(): exampletest_single('CSCI-4968-MBE/challenges/crackme0x03')
-def test_csci_4(): exampletest_single('CSCI-4968-MBE/challenges/crackme0x04')
-def test_csci_5(): exampletest_single('CSCI-4968-MBE/challenges/crackme0x05')
-def test_insomnihack_aeg(): exampletest_single('insomnihack_aeg')
-def test_android_license(): exampletest_single('android_arm_license_validation')
-def test_sym_write(): exampletest_single('sym-write')
-def test_angry_reverser(): exampletest_single('hackcon2016_angry-reverser')
-def test_sharif7(): exampletest_single('sharif7_rev50')
-def test_angrybird(): exampletest_single('codegate_2017-angrybird')
-@attr(speed='slow')
-def test_mbrainfuzz(): exampletest_single('secuinside2016mbrainfuzz') # 1m46s
-def test_unmapped_analysis(): exampletest_single('unmapped_analysis')
-@attr(speed='slow')
-def test_zwiebel(): exampletest_single('tumctf2016_zwiebel') # ~45m
-def test_csgames2018(): exampletest_single('csgames2018')
-def test_java_crackme1(): exampletest_single('java_crackme1')
-def test_java_simple3(): exampletest_single('java_simple3')
-def test_java_simple4(): exampletest_single('java_simple4')
-def test_ictf2017_javaisnotfun(): exampletest_single('ictf2017_javaisnotfun')
-def test_java_androidnative1(): exampletest_single('java_androidnative1')
-@attr(speed='slow')
-def test_defcon2019quals_veryandroidoso(): exampletest_single('defcon2019quals_veryandroidoso')
+def test_secconquals2016_ropsynth():
+    exampletest_single(
+        "secconquals2016_ropsynth"
+    )  # technically not that slow but impossible to run under multiprocessing
```

Comment:
You need to put this back on one line or the example_inclusion test won't be able to find it. just put the comment on its own line.